### ISSUE TITLE
next dev fails randomly on CI since 9.3.5 - exit code 137

### ISSUE THREAD

**Issue by pjaws** [vercel/next.js#11962]
# Bug report

## Describe the bug

Since upgrading to 9.3.5, `next dev` has failed with exit code 137 during our CircleCI builds about 20% of the time.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Update to 9.3.5
2. Run `next dev` on CircleCI
3. ?????
4. No profit 😞 

## Expected behavior

It would be nice if it didn't do this.

## Screenshots

```
> hi-next@0.3.1 dev /root/repo
> next dev

[ wait ]  starting the development server ...
[ info ]  waiting on ********************* ...
> Using external babel configuration
> Location: "/root/repo/babel.config.js"
[ ready ] compiled successfully - ready on *********************
[ wait ]  compiling ...
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

[ ready ] compiled successfully - ready on *********************
Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! hi-next@0.3.1 dev: `next dev`
npm ERR! Exit status 137
npm ERR! 
npm ERR! Failed at the hi-next@0.3.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-16T19_20_50_059Z-debug.log


Exited with code exit status 137
CircleCI received exit code 137
```

## System information

- OS: Debian (cypress/base:12.14.0) More info [here](https://github.com/cypress-io/cypress-docker-images/tree/master/base)
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: 9.3.5
- Version of Node.js: 12.14.0

## Additional Info

I haven't seen this at all when running locally.

**Comment by timneutkens** [comment]
Hmm 137 generally means an `Out of memory` type state where Node.js crashes 🤔 Would be happy to look into it but it'll be hard to reproduce without having the actual app to test against.

**Comment by pjaws** [comment]
@timneutkens Sorry, missed your comment. I know this doesn't give you much to work with in terms of reproducing the issue, but it's happening so frequently, I figured I'd at least report it and see if anyone else was experiencing this.

Currently testing a branch that reverts to 9.3.4 in CI to confirm it's related to Next.js/9.3.5.

**Comment by pjaws** [comment]
Looks like it's happening with 9.3.4 as well, so it must be something else that coincided with the update to 9.3.5. Sorry about that! Closing.

**Comment by pjaws** [comment]
Apologies again! While I was digging, I realized that we skipped 9.3.4 and upgraded from 9.3.3 to 9.3.5.

The first failing build with this error came immediately after the update, and I have tested a branch with 9.3.3 in CI repeatedly without issue.

So now, I'm 99% sure this is in some way related to Next.js 9.3.4 and reopening. Again, I wish I could provide more information or a way to reproduce it. I'll try to store the npm debug logs as artifacts from a failed build and see if they offer any additional info.

**Comment by Timer** [comment]
Closing as there was no reproduction given. Feel free to investigate this!

**Comment by manhcuongincusar1** [comment]
This 137 node happens when runs out of memory, almost vps with 1gb ram may get this issues. So you can create virtual ram for your VPS with "swap" on Linux. 
check it out here: https://linuxize.com/post/how-to-add-swap-space-on-ubuntu-20-04/

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
